### PR TITLE
Feat/harmonies & Some Cleanup

### PR DESCRIPTION
**Pull request by ThijsZijdel**
Screenshot 2024-08-06 at 15 31 35

### CODE REVIEW

File: src/Components/MonacoWrapper.tsx [hunk 1]
```diff
@@ -125,7 +151,10 @@ export const Monaco = ({
       width={width}
       options={{
         automaticLayout: true,
-        fontSize: 16,
+        fontSize: 12,
+        scrollBeyondLastLine: false,
```
Comment (Sparkenstein) [inline src/Components/MonacoWrapper.tsx:155]: I think scroll beyond last line is helpful espcially for diff-editor etc. but don't mind trying this one out too. lets see

File: src/Features/colors/ColorHarmonies.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      />
+
+      <Group gap={10} style={{ marginBottom: 14 }}>
+        <Text style={{ cursor: "pointer" }} fw="lighter" c="dimmed" size="sm">
+          Harmonies
+        </Text>
+      </Group>
+
+      {Object.keys(harmonies).map((key) => (
+        <RenderShades
```
Comment (Sparkenstein) [inline src/Features/colors/ColorHarmonies.tsx:54]: this will need `key` too.

File: src/Components/MonacoWrapper.tsx (hunk 1 above)
Comment (ThijsZijdel) [reply src/Components/MonacoWrapper.tsx:155]: Fair point, maybe enable a overwrite prop for over there?

**Review by Sparkenstein — APPROVED** [review]
Looks good to me, I will be doing some refactoring mainly:
1.  resuing some code
2. might refactor the class-component to function component but low priority
3. Will improve typings, I see a few any which are avoidable
4. Editor themes offline (high priority because devtools-x promises offline first)
5.

**Pull request merged**